tests/testFlagHandler.py

```diff
@@ -38,49 +39,98 @@
 from lsst.meas.base import FlagDefinition, FlagDefinitionVector, FlagHandler
 from lsst.meas.base.flagDecorator import addFlagHandler
 
-
+#   Configuration for Sample Plugin with a FlagHandler.
 class PythonPluginConfig(SingleFramePluginConfig):
 
-    failureType = lsst.pex.config.Field(dtype=int, default=None, optional=False,
-                                  doc="A failure mode to test")
+    edgeLimit = lsst.pex.config.Field(dtype=int, default=0, optional=False,
+                                  doc="How close to the edge can the object be?")
+    size = lsst.pex.config.Field(dtype=int, default=1, optional=False,
+                                  doc="size of aperture to measure around the center?")
+    flux0 = lsst.pex.config.Field(dtype=float, default=None, optional=False,
+                                  doc="Flux for zero mag, used to set mag if defined")
 
 @register("test_PythonPlugin")
-@addFlagHandler(("flag", "General Failure error"), ("flag_error1","First type of Failure occured."),
-                ("flag_error2", "Second type of failure occured."))
+
+@addFlagHandler(("flag", "General Failure error"),
+                ("flag_containsNan","Measurement area contains a nan"),
+                ("flag_edge", "Measurement area over edge"))
 class PythonPlugin(SingleFramePlugin):
     '''
-    This is a sample Python plugin.  The flag handler for this plugin is created
-    during construction, and is called using the method fail().  All plugins are
-    required to implement this method, which is used to set the flags in the
-    output source record if an error occurs.
+    This is a sample Python plugin which shows how to create and use a FlagHandler.
```

Comment:
Thanks for changing it. I'd have to check if single quotes are technically incorrect, but it's an overwhelmingly strong convention in Python. Always helps to be idiomatic in docs and examples.